src/components/FocusButton.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useRef } from 'react';
+
+import { Tooltip, Button } from 'antd';
+import { EyeTwoTone, EyeOutlined } from '@ant-design/icons';
+import PropTypes from 'prop-types';
+import {
+  useSelector, useDispatch,
+} from 'react-redux';
+
+import { setCellInfoFocus } from '../redux/actions/cellInfo';
+
+const FocusButton = (props) => {
+  const { store, lookupKey, experimentId } = props;
```

Comment:
`store` is a bit ambigious, better call it `focusType`

Reply:
it is called store because it determines what redux store the focus data is under. I made it explicit in the code now.

Reply:
I understand the rationale, but it is still a bit confusing when I see it in props like this, I am thinking that you are talking about the whole redux store. Maybe `focusStore` would be better?

Reply:
I changed it to `focusStore` in the code.